vnc_auto.html

```diff
@@ -80,7 +80,21 @@
                            "jsunzip.js", "rfb.js", "keysym.js"]);
 
         var rfb;
+        var resizeTimeout;
 
+
+        function UIresize() {
+            var innerWidth = window.innerWidth;
+            var innerHeight = window.innerHeight;
+            var controlBarHeight = $D('noVNC_status_bar').scrollHeight;
+            var borderHeight = 5;
```

Comment:
The noVNC_container element is for some reason a few pixels higher than the noVNC_canvas. It is 4 pixels higher in Chrome and 5 pixels higher in Firefox (according to the built in HTML inspectors). I have no idea why.. Me and a colleague spent half a day trying to figure this shit out but eventually gave up :unamused:

If someone has some insight into this I'd love to know!